cloud/openstack/keystone_user.py

```diff
@@ -252,8 +252,17 @@ def ensure_user_exists(keystone, user_name, password, email, tenant_name,
                                  email=email, tenant_id=tenant.id)
     return (True, user.id)
 
+def ensure_role_exists(keystone, role_name):
+    # Get the role if it exists
+    try:
+        role = get_role(keystone, role_name)
+    except KeyError:
+        # Role doesn't exist yet
+        role = keystone.roles.create(role_name)
+    return (True, role.id)
```

Comment:
I find it a little confusing that this method returns `(True, role.id)` if the role already exists. I would expect the boolean in the tuple to encode whether or not a change has happened.